docs/stdlib/set.rst

```diff
@@ -108,9 +108,9 @@ Sets
 
     :index: intersection
 
-    Test the membership of an element in a set.
+    Performs a logical check of membership for an element in a set.
```

Comment:
```suggestion
    Checks if a given element is a member of a given set.
```

We've stated this more simply in earlier documentation, and I'd like to keep consistent with that.